cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java

```diff
@@ -384,7 +384,11 @@ protected void loadNameMappings(String jsonStr) {
             LOGGER.debug("[nmi] Entity found: " + originalEntityId + ", " + originalEntityType);
 
             if (entityMapping.getNewEntityId() != null) {
-                newEntityId = entityMapping.getNewEntityId();
+            	LOGGER.debug("[nmi] IdPattern : " +entityMapping.getOriginalEntityIdPattern().toString());
```

Comment:
Indent in this block seem to be a bit weird... have you checked style (see https://github.com/telefonicaid/fiware-cygnus/blob/d3e4acd5942d858dfeeed4604de7c70e99c4d027/doc/contributing/contributing_guidelines.md#coding-style)?

Reply:
Fixed in caf2c82